domain/learningandworkprogress/src/test/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewScheduleCalculationServiceTest.kt

```diff
@@ -114,6 +114,7 @@ class ReviewScheduleCalculationServiceTest {
         "EXEMPT_PRISONER_TRANSFER",
         "EXEMPT_PRISONER_RELEASE",
         "EXEMPT_PRISONER_DEATH",
+        "EXEMPT_PRISONER_MERGE",
```

Comment:
I might see a merge conflict here when I get to raise a PR for my stuff, cos I'm refactoring this test!

He who merges last deals with the inevitable merge conflict! 😁